src/matcher/plugin.py

```diff
@@ -152,6 +153,17 @@ def _try_module_path(request: pytest.FixtureRequest) -> tuple[pathlib.Path, bool
     return (pathlib.Path(request.path.parent) / 'data' / 'expected', False)
 
 
+def _subst_pattern_parts(result: pathlib.Path, current: str, **kwargs: str) -> pathlib.Path:
+    # ATTENTION If pattern format started w/ `/`
+    # (or containing it in any place), just ignore it!
+    # ATTENTION Directory traversal also just SILENTLY ignored!
+    # TODO Or must be alerted?
```

Comment:
+1. Otherwise, it introduces implicit contradictory behavior. Maybe validate the ini-value for non-allowed parts before?

Reply:
I've added config validation and test for it.